typeset/start.js

```diff
@@ -28,13 +28,22 @@ const argv = yargs
     alias: 'f',
     describe: 'Output format for MathJax Conversion: html, svg. Default: html'
   })
+  .option('batch-size', {
+    alias: 'b',
+    describe: 'Number of math elements to convert as a batch. Default: 3000'
+  })
   .demandOption(['xhtml', 'output'])
   .help()
   .argv
 
 const pathToInput = path.resolve(argv.xhtml)
 const pathToCss = argv.css ? path.resolve(argv.css) : null
 let outputFormat = 'html'
+let batchSize = Number.isInteger(argv.batchSize) ? argv.batchSize : 3000
```

Comment:
You can use shorter syntax:
`let batchSize = Number(argv.batchSize) || 3000`

Reply:
Done :+1: